docs/Installing.asciidoc

```diff
@@ -336,6 +338,13 @@ required. This will direct all HTTP traffic to openQA.
 cp /etc/apache2/vhosts.d/openqa.conf.template /etc/apache2/vhosts.d/openqa.conf
 --------------------------------------------------------------------------------
 
+=== NGINX proxy
+
+The nginx config is already provided by the openQA RPM package.
+You only need to start the nginx server.
```

Comment:
That I would like to see a proof for because https://progress.opensuse.org/issues/131024#note-11 showed me something different :)